src/gui/touchscreengui.cpp

```diff
@@ -806,27 +729,28 @@ void TouchScreenGUI::handleReleaseEvent(int evt_id)
 void TouchScreenGUI::translateEvent(const SEvent &event)
 {
 	if (!m_visible) {
-		infostream << "TouchScreenGUI::translateEvent got event but not visible?!" << std::endl;
+		infostream
+			<< "TouchScreenGUI::translateEvent got event but not visible!"
+			<< std::endl;
 		return;
 	}
 
-	if (event.EventType != EET_TOUCH_INPUT_EVENT) {
+	if (event.EventType != EET_TOUCH_INPUT_EVENT)
 		return;
-	}
 
 	if (event.TouchInput.Event == ETIE_PRESSED_DOWN) {
-
-		/* add to own copy of eventlist ...
-		 * android would provide this information but irrlicht guys don't
+		/*
+		 * Add to own copy of event list...
+		 * android would provide this information but Irrlicht guys don't
 		 * wanna design a efficient interface
 		 */
-		id_status toadd;
+		id_status toadd{};
```

Comment:
Why this notation? They don't do anything. Please undo.

Reply:
This is required to initialise a class member

Reply:
So should I remove or leave? Theoretically, it does nothing, yes.
But IntelliJ IDEA's clang-tidy constantly swears when it's not there.

Reply:
My suggestion is to remove it.